packages/node_modules/@webex/plugin-meetings/src/locus-info/selfUtils.js

```diff
@@ -56,7 +56,8 @@ SelfUtils.parse = (self, deviceId) => {
       selfIdentity: SelfUtils.getSelfIdentity(self),
       selfUrl: self.url,
       removed: self.removed,
-      roles: SelfUtils.getRoles(self)
+      roles: SelfUtils.getRoles(self),
+      isUserUnadmitted: self.state === _IDLE_ && joinedWith?.intent?.type === _WAIT_,
```

Comment:
@arun3528,  Reason to not use `SelfUtils.isUserUnadmitted()` is when we pass `self` object from here, it doesn't contain `joinedWith` property which is being used inside the method.